### PR TITLE
MESH-1228: Rename `testnet-v1` to `testnet`

### PR DESCRIPTION
**Pull request by Centril**

### CODE REVIEW

**Comment by adamdossa** [comment]
@concourseatpolymath if the CODEOWNERS stuff is working correctly, this will need approval from yourself or @quinndiggitypolymath

**Review by maxsam4 — APPROVED** [review]

**Review by satyamakgec — APPROVED** [review]

**Pull request merged**
